### ISSUE TITLE
Firefox doesn't quit with Cmd+Q

### ISSUE THREAD

**Issue by thebitstick** [rbreaves/kinto#232]
I usually quit Firefox with Ctrl_L+Q, and since Cmd is Ctrl_R, all it does is close the window.
Is there any way I can make an exception for Firefox to quit. I don't want to change any other functionality.

**Comment by rbreaves** [comment]
Are you requesting this for Linux or Windows? Also I am confused at why you want it on Ctrl_L+A vs having a proper close all on Cmd+Q?

In either event I have now added the expected Cmd+Q == Ctrl+Q (close all windows, if browser) instead of Alt+F4 (single window close). This has been added to the Linux version, not the Windows version yet and it is on the dev branch.

https://github.com/rbreaves/kinto/commit/acd0b8031e25c2da4a29cc4b99d424af7e3e6dd6

**Comment by thebitstick** [comment]
Oh sorry, I mistyped. I meant Ctrl_L+Q.
And I am on Linux.

**Comment by rbreaves** [comment]
Well the issue should be resolved now in the dev branch if you want to try it out, unless you are really wanting to use the physical Ctrl_L key instead of the physical Alt/Cmd key location (key to the immediate left of the spacebar).

The reason the virtual Ctrl_R+Q key only closed a single instead is because by default it remaps to Alt+F4, I can't reliably expect all programs to respect the Ctrl+Q combo, but I suppose all popular browsers probably do, so that support has now been added. I guess I am not sure of the hotkey to close a single browser window instance under macOS, but I should probably added that too and support in Autohotkey, for Windows 10 support, as well before I merge this change into master.

Let me know if the dev branch as it currently is fixes the issue for you.

--------

On another note, for apps that do not support a "close all" feature.. I probably could write a shell script to help support such a feature, but I don't think I will be doing that right away. It would be as simple as writing a script to detect the active window though and then sending a quit or killall command to the apps process(es). If I did that though I would want to warn the user that all Windows will be closed in the same manner as the browsers do.
